### PR TITLE
 making the python script as windows executable file

### PR DESCRIPTION
**Pull request by Jaggi037**
## Description

created a run.bat file for windows users in a separate directory named windows_executable, windows users can simply run that file and use the service without worrying about any  dependencies.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Motivation and Context

i think, windows users after downloading the script may need to find out the dependencies which are required to run this script but after this change they don't have to worry about anything they just need to run a single file and everything will be done for them. 

## How Has This Been Tested?

-  i tested this particular feature on windows 11 as well as windows 10 systems
-  the testing of this feature was done on windows on which no dependencies were installed and also windows on which all the dependencies were installed and in both cases no issues were encountered 
- the change is in particularly separate directory so it does not affect any other part of the code 

## Checklist:

- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests to cover my changes.
- [x
[build_output.zip](https://github.com/ContriHUB/202020-Eyesafe-Project/files/12843148/build_output.zip)
] All new and existing tests passed.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

File: windows_executable/20-20-20-Eyesafe-project.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            t = threading.Timer(0, inner_wrap)
+            t.daemon = True
+            t.start()
+            return stop
+        return wrap
+    return outer_wrap
+
+@setInterval(1200,-1)
+def offandon(): 
+    #to turn off use :-
```
Comment (PriyavKaneria) [inline windows_executable/20-20-20-Eyesafe-project.py:33]: use the pass keyword for unimplemented or do nothing functions to prevent linting issues

**Review by PriyavKaneria — APPROVED** [review]
Looks good to me. It is a unique approach to solve the problem. Can you upload the generated dist as zip to this PR, I'll add it as a release in case someone wants to directly download?
Also I am not sure if that executable is portable i.e. can it be shared to someone and it works directly?

**Comment by PriyavKaneria** [comment]
Any updates on the suggestions? @Jaggi037

**Comment by Jaggi037** [comment]
hey @PriyavKaneria, the "20-20-20-Eyesafe-project.exe" file is inside the dist folder of the zip file which i had attached at the end of the submitted PR. 
i checked and verified that the windows executable file which is "20-20-20-Eyesafe-project.exe" file is supported by all windows users irrespective of their systems, means you can make a release of this file as a software for windows.
[build_output.zip](https://github.com/ContriHUB/202020-Eyesafe-Project/files/12853152/build_output.zip)

here i re-uploaded that "build_output.zip" file

**Comment by PriyavKaneria** [comment]
Got it. Can you make the change in function definition regarding the pass keyword mentioned in review.
Once done I can merge it

**Comment by Jaggi037** [comment]
hey @PriyavKaneria i dont think any function in this script needs pass keyword as no function has empty code blocks, can you please clarify your concern?

File: windows_executable/20-20-20-Eyesafe-project.py (hunk 1 above)
Comment (PriyavKaneria) [reply windows_executable/20-20-20-Eyesafe-project.py:33]: My bad the indentation was not visible in the github code review UI

**Review by PriyavKaneria — APPROVED** [review]
Github Code Review UI did not show indentations properly so got confused.

**Comment by PriyavKaneria** [comment]
Fixes #1

**Pull request merged**